src/main/java/brave/opentracing/BraveSpan.java

```diff
@@ -25,7 +25,7 @@
  * <p>This type also includes hooks to integrate with the underlying {@linkplain brave.Tracer}.
  * Ex you can access the underlying span with {@link #unwrap}
  */
-public final class BraveSpan implements Span {
```

Comment:
Tried to make BraveActiveSpan extend BraveSpan. Didn't end up liking the result and changed it back; forgot to clean this up. Will do!